src/compiler.adoc

```diff
@@ -1033,7 +1033,7 @@ Ran 1 tests containing 1 assertions.
 ----
 
 It is fine to test these kinds of assertions, but they are not very useful. Let's go to
-test some application code. For this, we will use a function to check if a year is leap
+test some application code. For this, we will use a function to check if a year is a leap year
```

Comment:
idiom: is leap -> is a leap year (leap not usually used as an adjective unless in front of a year, second, etc)